assets/tests/helpers/sentryInit.test.ts

```diff
@@ -1,21 +1,43 @@
-import { jest, test, expect } from "@jest/globals"
+import { jest, describe, test, expect } from "@jest/globals"
 import sentryInit from "../../src/helpers/sentryInit"
 import * as Sentry from "@sentry/react"
+import SentryFullStory from "@sentry/fullstory"
 
-jest.mock("@sentry/react", () => ({
-  __esModule: true,
-  init: jest.fn(),
-  setUser: jest.fn(),
-}))
+jest.mock("@sentry/react")
 
 const opts = {
   environment: "test_env",
 }
 
 const username = "test_username"
 
-test("Sentry Init runs", () => {
-  sentryInit(opts, username)
-  expect(Sentry.init).toHaveBeenCalledWith(opts)
-  expect(Sentry.setUser).toHaveBeenCalledWith({ username: username })
+const orgSlug = "test_org_slug"
+
+describe("sentryInit", () => {
+  test("Sentry.init runs with username", () => {
+    const mockedSentry = jest.mocked(Sentry)
+
+    sentryInit(opts, username)
+    expect(mockedSentry.init).toHaveBeenCalledWith(opts)
+    expect(mockedSentry.setUser).toHaveBeenCalledWith({ username: username })
+  })
+
+  test("Sentry.init runs without username", () => {
+    const mockedSentry = jest.mocked(Sentry)
+
+    sentryInit(opts)
+    expect(mockedSentry.init).toHaveBeenCalledWith(opts)
+    expect(mockedSentry.setUser).not.toHaveBeenCalled()
+  })
+
+  test("Sentry.init runs with organization slug to add FullStory integration", () => {
+    const mockedSentry = jest.mocked(Sentry)
+
+    sentryInit(opts, username, orgSlug)
+
+    const integrations = mockedSentry.init.mock.calls[0][0].integrations
+    if (integrations !== undefined && typeof integrations !== "function") {
+      expect(integrations[0]).toBeInstanceOf(SentryFullStory)
+    }
```

Comment:
suggestion(non-blocking): what do you think about this approach instead?
```suggestion
    expect(mockedSentry.init).toHaveBeenNthCalledWith(
      1,
      expect.objectContaining({
        integrations: expect.arrayContaining([expect.any(SentryFullStory)]),
      })
    )
```

Reply:
Oh, thanks, I meant to leave a comment but the way I initially did this was my least favorite part of the PR, and your suggestion is much better!